target/ppc/int_helper.c

```diff
@@ -1667,6 +1667,28 @@ void helper_xxinsertw(CPUPPCState *env, ppc_vsr_t *xt,
     *xt = t;
 }
 
+#define XXBLEND(name, sz) \
+void glue(helper_xxblendv, name)(ppc_avr_t *t, ppc_avr_t *a, ppc_avr_t *b,  \
+                                 ppc_avr_t *c, uint32_t desc)               \
+{                                                                           \
+    ppc_vsr_t tmp;                                                          \
+    int i;                                                                  \
+                                                                            \
+    for (i = 0; i < ARRAY_SIZE(t->glue(u, sz)); i++) {                      \
+        tmp.glue(u, sz)[i] = c->glue(s, sz)[i] >> (sz - 1);                 \
```

Comment:
Suggestion:
```
{
    int i;
    for (i = 0; i < ARRAY_SIZE(t->glue(u, sz)); i++) {
        t->glue(u, sz)[i] = (c->glue(u, sz)[i] >> (sz - 1)) ?
            b->glue(u, sz)[i] : a->glue(u, sz)[i];
    }
}
```

Reply:
Applied in https://github.com/PPC64/qemu/pull/19/commits/8ac8bd674cf161dae32893a66ef0f63539f3d643